converters/toZigbee.js

```diff
@@ -2047,8 +2048,9 @@ const converters = {
     danfoss_thermostat_orientation: {
         key: ['thermostat_orientation'],
         convertSet: async (entity, key, value, meta) => {
-            const payload = {'danfossThermostatOrientation': {value: (value ? 0x01: 0x00), type: 0x10}};
+            const payload = {'danfossThermostatOrientation': (value ? 1: 0)};
```

Comment:
Same since danfossThermostatOrientation is marked a bool, we can pass true/false

Reply:
Do we know when it is true and false?
e.g. true = vertical, false = horizontal? If we do, we might want to rename this thermostat_vertical_orientation instead... or horizontal... depending on which one results in a true value here.

Reply:
Found it in the pdf:
0x00: Horizontal (Default)
0x01: VerticalDefault is 0, but overwritten to value from production configuration at Init. 

It might be cleaner to add a new enum as mentioned https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592/files#r636517838

Reply:
on my Hive thermostat this command is not doing anything, unless I am missing something. The command works fine and the new value is reported back correctly, but the display does not turn. I wonder if this is Hive's firmware being dodgy. Settings this from the Hive app works though, but not until you turn the heating physically, only after that the display updates. Unfortunately I do not have an extra sniffer nor have I ever sniffed traffic so not sure how to approach. I am hoping that this will work for Danfoss Ally users though...

Reply:
saying that, I have just noticed a different attribute on a different cluster that might do just that. let me test
![image](https://user-images.githubusercontent.com/9019306/119129774-46dc3d00-ba2f-11eb-83df-73755914e428.png)

Reply:
@sjorge Yep, that did the trick. https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592/commits/47be7ab768405dcec0fe8ceda151c372217331af

Reply:
Looks like they are indeed 2 different things, one probably indicated if the valve is horizontal or vertical, the other which way the display is. E.g. when it's vertical you might still want to flip it for example if the TRV is in a corner.

Renaming `thermostat_orientation` to `thermostat_vertical_orientation` would make sense I think, as true indicates it's vertical and false horizontal. Or either we map true -> 'vertical' and false -> 'horizontal' and leave the name.

So the mqtt publish would be `{'thermostat_orientation': 'vertical'}` (true) or `{'thermostat_orientation': 'horizontal'}` (false)... but renaming the attribute seems easier and accomplishes the same effect?

Reply:
Bool Solved here
https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592/commits/5338a8efcec58d1018d252f62bb519ace285fefd

thermostat_vertical_orientation makes the most sense
Solved here
https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592/commits/270708f40cb87a08b384d97d1a7cc2eb066187ed